content/import-metadata.mdx

```diff
@@ -4,16 +4,17 @@ import SpectraTableFooterLinks from '/components/SpectraTableFooterLinks';
 
 export const imagePath = '/images/spectra-table/';
 
-The Import Metadata feature allows users to add metadata to their existing spectra by uploading a CSV file containing the metadata information. This provides flexibility in associating relevant data with the spectra, enabling better organization and analysis.
+The Import Metadata feature allows users to add metadata to their existing spectra by uploading a plain text file (**.csv** or **.txt** ) containing the metadata information. This provides flexibility in associating relevant data with the spectra, enabling better organization and further analysis.
 
-### CSV File Format
+### Plain Text File Format
 
-The CSV file should adhere to the following format:
+The file should adhere to the following format:
 
 - The first row should contain the column headers, which can have any names chosen by the user.
 - Subsequent rows represent individual records or entries.
-- Fields should be separated by one valid delimiter: comma or semicolon.
-- If a field contains commas, semicolon or special characters, it should be enclosed in double quotes.
+- Decimal separators should be either point (.) or comma (,)
+- Fields should be separated by one valid delimiter: comma ( , ), semicolon ( ; ), pipe ( | ), space or tab ( ).
```

Comment:
To avoid confusion between delimiter and separator.
```suggestion
- Fields should be delimited by one of these characters: comma ( , ), semicolon ( ; ), pipe ( | ), space or tab.
```

Reply:
Changed